Open Judge System/OJS.Common/Models/ExecutionStrategyType.cs

```diff
@@ -24,12 +31,6 @@ public enum ExecutionStrategyType
         SqlServerLocalDbRunSkeletonRunQueriesAndCheckDatabase = 15,
         MySqlPrepareDatabaseAndRunQueries = 16,
         MySqlRunQueriesAndCheckDatabase = 17,
-        MySqlRunSkeletonRunQueriesAndCheckDatabase = 18,
-        CSharpUnitTestsExecutionStrategy = 23,
-        CSharpProjectTestsExecutionStrategy = 24,
-        JavaProjectTestsExecutionStrategy = 25,
-        CPlusPlusZipFileExecutionStrategy = 26,
-        JavaUnitTestsExecutionStrategy = 27,
-        CSharpAspProjectTestsExecutionStrategy = 28
+        MySqlRunSkeletonRunQueriesAndCheckDatabase = 18
```

Comment:
Strategies are now ordered by language in groups (makes it more readable and easier to find what you're looking for). I looked online and found out that such orderings are common as they bring semantic benefits.